sekizai/templatetags/sekizai_tags.py

```diff
@@ -82,6 +94,8 @@ def render_tag(self, context, name, postprocessor, nodelist):
         if postprocessor:
             func = import_processor(postprocessor)
             data = func(context, data, name)
+        elif postprocessor is None and name in _mapped_postprocessors:
```

Comment:
Because by checking for None I can distinguish between postprocessor not set, or intentionally deactivating the postprocessor for this templatetag using an empty string. This is part of the docs.